### PR TITLE
Old history stuff

### PR DESCRIPTION
**Pull request by Malicos**
## Changelog:
- Ported leftover stuff from old 2021 GW branch.

## Developer changelog:
- Added e_high_khanate, to be used in 508 startdate.
- Stormsong is now known as Briarback for quilboar holders.
- Added flavor for Barathen III and his wife, Varia.
- Barathen the Adamant is now correctly Barathen III the Adamant, as there are now two preceding Barathens.
- Expanded Atal'ai into Black Morass in pre-583 bookmarks.
- Strand of the Ancients is now known by it's WotLK Alpha name, Alduran
- Added Shadeglen troll cultural names to Silverpine Forest, to be used in 508 startdate.

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
TBA

### CODE REVIEW

File: localization/replace/english/wc_title_replacements_l_english.yml [hunk 1]
```diff
@@ -229,4 +229,10 @@
...
  b_larne:0 "Lar'ne"
- b_carrion:0 "Carrion"
\ No newline at end of file
+ b_carrion:0 "Carrion"
+ 
+ d_strand_of_ancients:0 "Alduran"
+ d_strand_of_ancients_adj:0 "Aldurani"
+
+ d_stormsong:0 "Briarback"
+ d_stormsong_adj:0 "Briarback"
```
Comment (Grekopithikos) [inline localization/replace/english/wc_title_replacements_l_english.yml]: So why is Stormsong called Briarback when not controlled by humans?
Shouldn't it be called Briarback only when controlled by quillboars?

File: history/characters/1_azerothian.txt [hunk 2]
```diff
@@ -4,15 +4,23 @@
...
 	trait=education_martial_3 trait=education_martial_prowess_3 trait=brave trait=trusting trait=generous
 	father=7
 	disallow_random_traits = yes
-	490.1.1 = { birth=yes }
+	490.1.1 = { 
+		birth = yes trait = creature_human
+		effect = { make_important_lore_character_effect = yes } 
+	}
+	507.7.16 = {
+		add_spouse = 3	#Varia
```
Comment (Grekopithikos) [inline history/characters/1_azerothian.txt:18]: Would they have been soulmates?

Comment (Malicos) [reply history/characters/1_azerothian.txt:18]: added

File: localization/replace/english/wc_title_replacements_l_english.yml (hunk 1 above)
Comment (Malicos) [reply localization/replace/english/wc_title_replacements_l_english.yml]: fixed

File: common/landed_titles/01_landed_titles.txt [hunk 3]
```diff
@@ -25467,6 +25492,16 @@ e_lordaeron = {
...
+				
+				cultural_names = {
+					name_list_amani = cn_shadeglen
+					name_list_blood_troll = cn_shadeglen
+					name_list_dark_troll = cn_shadeglen
+					name_list_drakkari = cn_shadeglen
+					name_list_farraki = cn_shadeglen
+					name_list_gurubashi = cn_shadeglen
+					name_list_zandalari = cn_shadeglen
+				}
```
Comment (Grekopithikos) [inline common/landed_titles/01_landed_titles.txt:25504]: Why would both c_ferry and c_undernelly use the same cultural name?

File: history/titles/00_k_stormwind.txt [hunk 4]
```diff
@@ -2,18 +2,18 @@
...
-	303.8.14={holder=60} #Carrington[1]
+	303.8.14={holder=60} #Barathen[1]
 	326.9.2={holder=63} #Angus[1]
 	350.12.6={holder=70} #Barrett[1]
 	370.3.18={holder=73} #Hyatt[1]
 	386.4.13={holder=76} #Acton[1]
 	425.7.27={holder=80} #Adam[1]
 	456.1.30={holder=7} #Landan[1]
-	506.1.1={holder=1} #Barathen[1]
+	504.2.13={holder=1} #Barathen[1]
```
Comment (Grekopithikos) [inline history/titles/00_k_stormwind.txt:16]: Why would he inherit the kingdom 2 years before Landan's death?

File: common/landed_titles/01_landed_titles.txt (hunk 3 above)
Comment (Malicos) [reply common/landed_titles/01_landed_titles.txt:25504]: but they don't? Ferry is Zeb'rokh. Undernelly is Shadeglen.

File: history/titles/00_k_stormwind.txt (hunk 4 above)
Comment (Malicos) [reply history/titles/00_k_stormwind.txt:16]: now dies correctly in 504.

File: common/landed_titles/01_landed_titles.txt (hunk 3 above)
Comment (Grekopithikos) [reply common/landed_titles/01_landed_titles.txt:25504]: oh sorry I meant c_dawning_isles

Comment (Malicos) [reply common/landed_titles/01_landed_titles.txt:25504]: Dawning Isles don't have a cn. Duchy of Lordamere is Shadeglen (d). County of Ferry is Zeb'rokh. County of Undernelly is Shadeglen (c). County of Valganfield is Shad'watha.

File: history/characters/1_azerothian.txt [hunk 5]
```diff
@@ -61,8 +72,14 @@
...
 	trait=education_diplomacy_3 trait=chaste trait=patient
 	disallow_random_traits = yes
-	502.6.9 = { birth=yes }
-	587.1.1 = { death=yes }
+	496.6.9 = { 
+		birth=yes trait=creature_human 
+		effect = { make_important_lore_character_effect = yes } 
+	}
+	507.7.16 = {
+		add_spouse = 1	#Barathen
```
Comment (Grekopithikos) [inline history/characters/1_azerothian.txt:81]: Also I think Varia should be heterosexual as well if Barathen is

Comment (Malicos) [reply history/characters/1_azerothian.txt:81]: fixed

File: localization/replace/english/wc_title_replacements_l_english.yml [hunk 6]
```diff
@@ -230,6 +230,9 @@
  b_ronda:0 "Ron'da"
  b_larne:0 "Lar'ne"
  b_carrion:0 "Carrion"
- 
+
+ d_strand_of_ancients:0 "Alduran"
+ d_strand_of_ancients_adj:0 "Aldurani"
```
Comment (Grekopithikos) [inline localization/replace/english/wc_title_replacements_l_english.yml:235]: why has this been renamed?

Comment (Malicos) [reply localization/replace/english/wc_title_replacements_l_english.yml:235]: it's an actual name of the place according to wotlk alpha, Strand of Ancients sounds generic

**Review by Grekopithikos — APPROVED** [review]
LTGM!

**Pull request merged**